experiment/dispatcher.py

```diff
@@ -195,7 +195,7 @@ def main():
     except Exception as error:
         logs.error('Error conducting experiment.')
         raise error
-    experiment_config_file_path = os.path.join(_get_config_dir(),
+    experiment_config_file_path = os.path.join(_get_config_path(),
```

Comment:
this looks wrong, shouldnt it be just just _get_config_path() and not experiment.yaml

Reply:
Fixed.